cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
```

Comment:
The `validate_ticket` method seems to be using `ValidationError` incorrectly. It should be used to raise an exception if the validation fails, rather than being passed as an argument. Ensure that `validate_ticket` is implemented to raise `ValidationError` when necessary.